### ISSUE TITLE
func executable resolution difficulties

### ISSUE THREAD

**Issue by alastairs** [corvus-dotnet/Corvus.Testing#397]
As also noted in #253, there are several inconsistencies around where the Node toolchain will place packages and therefore the `func` executable. In my own case, using [volta.sh](https://volta.sh) as my node version manager, packages are located separately from the Node environment:

```plain
❯ npm prefix -g
%LOCALAPPDATA%\Volta\tools\image\node\16.18.0
❯ npm root -g
%LOCALAPPDATA%\Volta\tools\image\node\16.18.0\node_modules
❯ volta which func
%LOCALAPPDATA%\Volta\tools\image\packages\azure-functions-core-tools\func
```

This obviously prevents any Corvus tests from running. What does resolve the correct path, however, is the `where` utility executable:
```plain
❯ where.exe func
%LOCALAPPDATA%\Volta\bin\func
%LOCALAPPDATA%\Volta\bin\func.exe
```

Perhaps using `which` on *nix systems and `where.exe` on Windows would be a more reliable method of determining the installation path for this tool?

https://github.com/corvus-dotnet/Corvus.Testing/blob/31434098d3018bf7533c0b360049fca3a69660f2/Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/FunctionsController.cs#L288

**Comment by mwadams** [comment]
That's not a bad idea. If you want to submit a PR we will expedite it.

**Comment by idg10** [comment]
Fixed by #398

**Comment by mwadams** [comment]
"For certain values of expedite".

Sorry @alastairs - that slipped through the net!
